packages/app/src/breadcrumbs/BreadcrumbsBar.tsx

```diff
@@ -66,6 +79,7 @@ function BreadcrumbsBar(props: Props) {
           icon={FiMessageCircle}
           href="/" // replaced dynamically
           target="_blank"
+          style={{ marginLeft: '0.375rem' }}
```

Comment:
This spaces out the feedback button a little. Adding a class was a bit tricky, so I didn't bother.

Reply:
You could rearrange the buttons with a separator instead:

![image](https://user-images.githubusercontent.com/42204205/161210262-73d1da0a-bf0c-4643-9bf9-5c8a425a74ff.png)